### PR TITLE
Add Test env configuration for Web portal

### PR DESCRIPTION
**Pull request by LallanKumarYadav**
Add Test env configuration for Web portal,
Need to validate properties file for test-private ts.

### JIRA Ticket

SDC-2464

### Summary
Add Test env configuration for Web portal,
========  Validate properties file for test-private ts.  ==========

### CODE REVIEW

File: webportal/webapp/test_deploy.sh [hunk 1]
```diff
@@ -1,6 +1,19 @@
 #/bin/bash
-ng build --aot=true --configuration=test
-#Copy everything over to S3 bucket
-aws s3 cp --profile sdc ./dist s3://test-dot-sdc-webportal-hosting --recursive --metadata-directive REPLACE --cache-control max-age=86400 --acl public-read
-#Bust open the cache
-aws s3 cp --profile sdc ./dist/index.html s3://test-dot-sdc-webportal-hosting/index.html --region us-east-1 --metadata-directive REPLACE --cache-control max-age=0 --acl public-read
+
```
Comment (TiannaW24) [inline webportal/webapp/test_deploy.sh:2]: Are we replacing the test environment with the test-private environment? If so, you can delete the configuration in the angular.json file. If not, you should be creating a new test_private_deploy.sh script.

Comment (LallanKumarYadav) [reply webportal/webapp/test_deploy.sh]: That makes sense, I have added a separate deploy script for test deploy env.

File: webportal/webapp/test_private_deploy.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+echo "***"
+echo "Building assets..."
+ng build --configuration=test-private
+
+
+# Use distinct test-private bucket so we can modify URLs, etc. per the environment
+# Copy everything over to S3 bucket
+echo "Copying to s3..."
+aws s3 cp --profile sdc ./dist s3://test-private-sdc-webportal-hosting --recursive --only-show-errors
```
Comment (aaronFranssell) [inline webportal/webapp/test_private_deploy.sh:11]: Did this bucket already get created?

Comment (LallanKumarYadav) [reply webportal/webapp/test_private_deploy.sh:11]: > Did this bucket already get created? If so, is it in Terraform somewhere?

I created this bucket manually. I couldn't find anywhere in terraform.

Comment (JAMSUPREME) [reply webportal/webapp/test_private_deploy.sh:11]: Likewise for the other temporary s3 private buckets. Might be worth adding the bucket creation into webportal terraform once we add some of the other pre-reqs.

**Pull request merged**